README.md

```diff
@@ -8,7 +8,7 @@ The blog was created to help all students practice their Git, GitHub skills and
 Yes, of course. The blogs purpose is for students to fork the repository, make changes in the required areas, commit them and make a pull request which the repository owner will review. These changes will be visible on the website once accepted.
 
 # Where Can I Make Changes?
-Every student can make changes to the 3 tabs of 'Learning Git', 'Exploring AWS', 'Implementing CloudFormation' and on the 'Student Blogs' page.
+Every student can make changes to the 3 tabs of 'Git and GitHub', 'Understanding the cloud with AWS', 'CloudFormation with AWS' and on the 'Student Blogs' page.
```

Comment:
Hi AgbJosh, 

If you don't mind me asking why did you want to change the tab names in the readme file to match the panel headings on the home tab? Were you suggesting to change the actual tab names at the top?

Reply:
Hi, in the README, you mentioned the tab names under which other students could add their comments. However, the tab names on the web page have slightly different headings than the one in the README. I thought it could improve the user experience if the tab names in both the README and on the page were the same.

Reply:
Thanks for your response. 

Just so I know I'm not misunderstanding the change, could you please see the image below.
![Screenshot 2019-12-21 at 14 54 50](https://user-images.githubusercontent.com/59033909/71309641-51da0a80-2402-11ea-9298-091abf4d6119.png)

Reply:
Sorry.....my bad. I didn't scroll up to see. I believed it was referring to the other headings in the content. Ignore my changes.

Reply:
That's perfectly ok and not a problem.

Congratulations on being the first to commit and contribute to this project!